roles/nodepool/tasks/main.yml

```diff
@@ -93,8 +106,22 @@
   synchronize:
     src: files/etc/nodepool/
     dest: /etc/nodepool/
-    rsync_opts:
-      - "--chown=nodepool:nodepool"
+# The option below is amazing, but requires rsync 3.1+
+#    rsync_opts:
+#      - "--chown=nodepool:nodepool"
+#
+# So we need to manually do the same thing after the fact
+
+- name: Get list of files in /etc/nodepool
+  command: find /etc/nodepool -type f \! -name nodepool.yaml -a \! -name secure.conf
+  register: files_in_etc_nodepool
```

Comment:
why aren't nodepool.yaml and secure.conf owned by nodepool anyway?

Reply:
Feels like there is a better way we could organize all this stuff, maybe put nodepool templates in opt?